pkg/controller/backup_controller.go

```diff
@@ -398,6 +399,13 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 	// Otherwise, the JSON file in object storage has a CompletionTimestamp of 'null'.
 	backup.Status.CompletionTimestamp.Time = c.clock.Now()
 
+	backup.Status.VolumeSnapshotsAttempted = len(backup.VolumeSnapshots)
```

Comment:
Any interest in sending these numbers to prometheus, aggregated by schedule?

Reply:
@nrb how about I add a separate issue for it and we leave this as-is?